test/integ/routers/alpha-router/alpha-router.integration.test.ts

```diff
@@ -844,6 +891,91 @@ describe('alpha router integration', () => {
       });
     });
   }
+
+  describe('Mixed routes', () => {
+    const tradeType = TradeType.EXACT_INPUT;
+
+    const BOND_MAINNET = new Token(
+      1,
+      '0x0391D2021f89DC339F60Fff84546EA23E337750f',
+      18,
+      'BOND',
+      'BOND'
+    );
+
+    const APE_MAINNET = new Token(
+      1,
+      '0x4d224452801aced8b2f0aebe155379bb5d594381',
+      18,
+      'APE',
+      'APE'
+    );
+
+    beforeAll(async () => {
+      await hardhat.fund(
+        alice._address,
+        [parseAmount('10000', BOND_MAINNET)],
+        [
+          '0xf510dde022a655e7e3189cdf67687e7ffcd80d91', // BOND token whale
+        ]
+      );
+      const aliceBONDBalance = await hardhat.getBalance(
+        alice._address,
+        BOND_MAINNET
+      );
+      expect(aliceBONDBalance).toEqual(parseAmount('10000', BOND_MAINNET));
+    });
```

Comment:
I think we should just add a `forceMixedRoutes` flag for testing instead of this, over time this token combo might start returning v3 only routes or something

Reply:
Addressed in #158